menu/menu.yaml

```diff
@@ -184,6 +184,8 @@
         Title: Version 1 to 2
       - Url: nservicebus/upgrades/gateway-2to3
         Title: Version 2 to 3
+      - Url: nservicebus/upgrades/gateway-3to3.1
```

Comment:
do we need an upgrade guide? The new API will be mentioned in the 3-4 upgrade guide when the old API will be deprecated I assume?

Reply:
We have other upgrade guides for minors too:

- https://docs.particular.net/nservicebus/upgrades/6to6.2
- https://docs.particular.net/nservicebus/upgrades/metrics-1to1.1
- https://docs.particular.net/transports/upgrades/asb-9to9.1
- https://docs.particular.net/transports/upgrades/rabbitmq-4.0to4.1
- https://docs.particular.net/persistence/upgrades/sql-4.2to4.3

Reply:
Removed upgrade guide as discussed